### PR TITLE
Add Py 1.0

### PR DESCRIPTION
**Pull request by zshipko**

### CODE REVIEW

**Comment by zshipko** [comment]
It looks like `m4` is missing on Alpine, or is this something I need to fix on my end?

**Comment by mseri** [comment]
I think it's more general, plenty of packages are failing in that way

File: packages/py/py.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+available: [ ocaml-version >= "4.03.0"]
+depends:
+[
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "topkg" {build & >= "0.9.1"}
+  "ctypes" {>= "0.13.0"}
+  "ctypes-foreign" {>= "0.4.0"}
+]
+depopts: []
```
Comment (mseri) [inline packages/py/py.1.0/opam]: You can remove this line

**Comment by mseri** [comment]
I can see on the repository that you have tests but you don't have a `build-test` secion in the `opam ` file to run them in the CI. Probably it's worth adding it

**Comment by zshipko** [comment]
Okay, @mseri - I have made your suggested changes. Please let me know if there is anything else I should address before it's ready to merge.

Thanks.

**Comment by mseri** [comment]
Thanks! The tests now are failing. I think it's due to the missing python devel libraries. You should be able to fix it in most cases adding specific `depext`s to the `opam` file. I think the following should be enough

```
depexts: [
  [["debian"] ["python3-dev"]]
  [["ubuntu"] ["python3-dev"]]
  [["fedora"] ["python3-devel"]]
  [["centos"] ["python3-devel"]]
  [["alpine"] ["python3-dev"]]
]
```

**Comment by zshipko** [comment]
@mseri, Thanks! I have updated the depexts.

**Comment by mseri** [comment]
Mmh, it’s a bit trickier than that. The error now says

```
libpython3.4m.so: undefined symbol: Py_DecodeLocale
```

And from python’s documentation looks like this symbol has been introduced in python 3.5
I don’t think there is an easy way to restrict the depext to python > 3.5, I am not sure if you can/want to do anything about that.

**Comment by zshipko** [comment]
Hmm, yeah I saw that. I don't think I want to support Python 3.4 right now. Thanks for your help.

**Comment by hcarty** [comment]
@zshipko It's ok to only support Python 3.5. It may be better to leave out the depext for OS distributions which don't provide a recent enough Python version and include a note stating that Python 3.5 or later is required.

**Comment by zshipko** [comment]
@hcarty, Great! I've documented the requirement in the descr file. Is that sufficient?

**Comment by hcarty** [comment]
You could add some extra messages for users (see https://github.com/ocaml/opam-repository/blob/master/packages/lwt/lwt.3.2.1/opam and https://github.com/ocaml/opam-repository/blob/master/packages/mysql/mysql.1.2.2/opam for examples). I think this is ok to merge as-is or with the extra messages. What is your preference @zshipko?

**Comment by zshipko** [comment]
@hcarty I just added a message to the opam file.

**Comment by mseri** [comment]
I think it’s fine to merge now. I’ll leave it open for comments a bit more and merge in the afternoon

**Comment by zshipko** [comment]
@mseri Sounds good!

**Comment by hcarty** [comment]
Agreed that this looks good. Thank you for your patience @zshipko!

**Comment by mseri** [comment]
Thanks for the patience. Feel free to announce this release on discuss.ocaml.org, we have an `Announce` tag and a `Community` category (or maybe the other way around) for this purpose

**Pull request merged**